TestAssets/TestProjects/EndToEndTestApp/project.json

```diff
@@ -20,11 +20,12 @@
     "files": {
       "includeFiles": "packfiles/pack1.txt",
       "mappings": {
-        "newpath/": "packfiles/pack2.txt"
+        "newpath/": "packfiles/pack2.txt",
+        "anotherpath/": "packfiles/pack2.txt"
       }
     }
   },
-  "publishOptions": "testpublishfile.txt",
+  "publishOptions": [ "testpublishfile.txt", "publishfiles" ],
```

Comment:
Do we still have a test for 

```
"publishOptions": "singleFile.ext"
```

?

I think both scenarios need to be covered.